### PR TITLE
[AN-4348] Adds new image preview to image sharing

### PR DESCRIPTION
**Pull request by shumengye**
#### Description
Unifies image sharing preview to use new image preview. This fixes previously wrong image scale type and missing dark backgrounds on title and confirmation buttons.

#### Ticket
https://wearezeta.atlassian.net/browse/AN-4348
#### APK
[Download build #7276](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7276/artifact/build/artifact/wire-dev-PR26-7276.apk)
[Download build #7280](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7280/artifact/build/artifact/wire-dev-PR26-7280.apk)
[Download build #7312](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7312/artifact/build/artifact/wire-dev-PR26-7312.apk)
[Download build #7316](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7316/artifact/build/artifact/wire-dev-PR26-7316.apk)
[Download build #7323](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7323/artifact/build/artifact/wire-dev-PR26-7323.apk)
[Download build #7324](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7324/artifact/build/artifact/wire-dev-PR26-7324.apk)
[Download build #7326](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7326/artifact/build/artifact/wire-dev-PR26-7326.apk)

### CODE REVIEW

**Comment by hakonbo** [comment]
build again

File: app/src/main/java/com/waz/zclient/pages/main/profile/camera/CameraFragment.java [hunk 1]
```diff
@@ -393,26 +393,26 @@ private void showPreview(ImageAsset imageAsset, boolean bitmapFromCamera) {
...
-                                                CursorImagesPreviewLayout.Source.CAMERA,
-                                                this,
-                                                getControllerFactory().getAccentColorController().getAccentColor().getColor(),
-                                                previewTitle);
-
-        imagePreviewContainer.addView(cursorImagesPreviewLayout);
+            getStoreFactory().getConversationStore().getCurrentConversation().getName() :
+            "";
+        imagePreviewLayout.setImageAsset(imageAsset,
+            ImagePreviewLayout.Source.CAMERA,
```
Comment (hakonbo) [inline app/src/main/java/com/waz/zclient/pages/main/profile/camera/CameraFragment.java]: Indentation is not correct, please copy ```codeStyleSettings.xml``` into ```.idea``` folder, restart and reformat.

**Comment by hakonbo** [comment]
lgtm, just need to update to follow our codestyle...

File: app/src/main/java/com/waz/zclient/pages/main/profile/camera/CameraFragment.java (hunk 1 above)
Comment (shumengye) [inline app/src/main/java/com/waz/zclient/pages/main/profile/camera/CameraFragment.java]: done

**Comment by shumengye** [comment]
@hakonbo my preferences got changed after switching to wire-android..fixed

**Pull request merged**